### PR TITLE
Refactor Result Code

### PR DESCRIPTION
**Pull request by wadethestealth**
# ♻️ Clean Up Main
Move result code out of main, and pass store as an arg.

## To-Do:
- [x] add JsDoc comments

### CODE REVIEW

File: lib/main.js [hunk 1]
```diff
@@ -423,7 +327,10 @@ const Hydrogen = {
       if (moveDown === true) {
         codeManager.moveDown(editor, row);
       }
-      this.createResultBubble(editor, { code, row, cellType });
+      this.checkForKernel(store, kernel => {
+        //$FlowFixMe
+        result.createResult(store, { code, row, cellType });
```
Comment (wadethestealth) [inline lib/main.js]: I can't fix this flow error for the life of me. It is also in 3 other places

Comment (aviatesk) [reply lib/main.js]: Will try to fix this in next week or so.

Comment (aviatesk) [reply lib/main.js]: Fixed in a1877f2 :)

**Comment by aviatesk** [comment]
Gonna merge this if @wadethestealth gets satisfied with my changes.

**Comment by wadethestealth** [comment]
Looks good @aviatesk

**Pull request merged**